api/client/src/test/java/org/projectnessie/client/auth/oauth2/TestOAuth2Client.java

```diff
@@ -581,7 +581,7 @@ private void handleTokenEndpoint(HttpServletRequest req, HttpServletResponse res
         throws IOException {
       soft.assertThat(req.getMethod()).isEqualTo("POST");
       soft.assertThat(req.getContentType()).isEqualTo("application/x-www-form-urlencoded");
-      soft.assertThat(req.getHeader("Authorization")).isEqualTo("Basic QWxpY2U6czNjcjN0");
+      soft.assertThat(req.getHeader("Authorization")).isEqualTo("Basic Q2xpZW50MTpzM2NyM3Q=");
```

Comment:
Is this change caused by the change to `configBuilder` below? (Just reviewing in a browser, not an IDE)

Reply:
See line 843 below. I changed `.clientId("Alice")` to `.clientId("Client1")` because the old client name was misleading. But that changed the Base 64 hash.